src/ert/scheduler/slurm_driver.py

```diff
@@ -384,17 +398,47 @@ async def _poll_once_by_scontrol(
             )
             return None
 
-        info = None
         try:
-            info = _parse_scontrol_output(stdout.decode(errors="ignore"))
+            return _parse_scontrol_output(stdout.decode(errors="ignore"))
         except Exception as err:
             logger.error(
-                f"Could no parse scontrol stdout {stdout.decode(errors='ignore')}: {err}"
+                f"Could not parse scontrol stdout {stdout.decode(errors='ignore')}: {err}"
             )
-            return info
-        self._scontrol_cache[missing_job_id] = info
-        self._scontrol_cache_timestamp = time.time()
-        return info
+        return None
+
+    async def _run_sacct(self, missing_job_id: str) -> Optional[ScontrolInfo]:
+        try:
+            process = await asyncio.create_subprocess_exec(
+                self._sacct,
+                "-X",
+                "-n",
+                "-o",
+                "State,ExitCode",
+                "-P",
+                "-j",
+                str(missing_job_id),
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.PIPE,
+            )
+        except FileNotFoundError:
+            logger.error("sacct binary not found")
+            return None
+        stdout, stderr = await process.communicate()
+        if process.returncode:
```

Comment:
`sacct` on my ad-hoc slurm cluster always returns exit code 1 and "Slurm accounting storage is disabled", it seems to be correctly handled here 👍🏻